sklearn/ensemble/gradient_boosting.py

```diff
@@ -850,25 +850,26 @@ def _check_params(self):
             if self.max_features == "auto":
                 # if is_classification
                 if self.n_classes_ > 1:
-                    max_features = max(1, int(np.sqrt(self.n_features)))
+                    max_features = max(1, int(np.sqrt(self._n_features)))
```

Comment:
because before `n_features` was changed by `fit` here: https://github.com/kiote/scikit-learn/blob/0e18dc9141ee97e6e183b5b457f9670e17c8fe8b/sklearn/ensemble/gradient_boosting.py#L981

and now that makes tests to fail with the new checker.